File: common-core/src/main/java/com/ityu/dao/system/CfgRepository.java

package com.ityu.dao.system;


import com.ityu.bean.entity.system.Cfg;
import com.ityu.dao.BaseRepository;

/**
 * 全局参数dao
 *
 * @author ：enilu
 * @date ：Created in 2019/6/29 12:50
 */
public interface CfgRepository extends BaseRepository<Cfg,Long> {

    Cfg findByCfgName(String cfgName);
}

File: common-core/src/main/java/com/ityu/common/dao/system/CfgRepository.kt
package com.ityu.common.dao.system

import com.ityu.common.bean.entity.system.Cfg
import com.ityu.common.dao.BaseRepository

/**
 * 全局参数dao
 *
 * @author ：enilu
 * @date ：Created in 2019/6/29 12:50
 */
interface CfgRepository : BaseRepository<Cfg?, Long?> {
    fun findByCfgName(cfgName: String?): Cfg?
}
